tests/integration/courses.test.js

```diff
@@ -4134,6 +4134,44 @@ describe('COURSES ROUTES - DELETE /courses/{_id}/companies{companyId}', () => {
     });
```

Comment:
Ligne 4072, il faut que tu modifies l'intitulé du test :     it('should return a 403 if course is INTRA , async () => {

Reply:
+ ETQ ROF tu n'as pas ajouté de test pour une formation intra_holding